.github/workflows/tests.yaml

```
@@ -13,3 +13,66 @@ jobs:
       - run: pip3 install poetry
       - run: poetry install --with test
       - run: poetry run pytest
+
+  integration-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+      - name: Install python dependencies
+        run: |
+          pip3 install apache-airflow
+          pip3 install .
+      - name: Start kind cluster
+        uses: helm/kind-action@v1.4.0
+      - name: Verify cluster up
+        run: |
+          kubectl cluster-info
+          kubectl get nodes
+      - name: Setup cluster for tests
+        run: |
+          kubectl create sa airflow
+          kubectl create secret generic github-app-secret --from-literal=test=test
+          kubectl create cm ca-bundle-pem --from-literal=test=test
+      - name: Setup Airflow
+        env:
+          AIRFLOW_CONN_SQLITE_DEFAULT: sqlite://?mode=ro
+        run: |
+          airflow db reset -y
```

Skulle vi lagt til `--skip-init`?
```suggestion
          airflow db reset --skip-init --yes
```

---

Må initialiseres for å kunne sette variabler i databasen.